tinytuya/Cloud.py

```diff
@@ -481,7 +481,7 @@ def getdps(self, deviceid=None):
             )
         return response_dict
 
-    def sendcommand(self, deviceid=None, commands=None):
+    def sendcommand(self, deviceid=None, commands=None, uri='iot-03/devices/'):

```

Comment:
Looks clean and a fairly safe way to added additional customization (uri).  

@uzlonewolf any concerns?